draft-ietf-taps-arch.md

```diff
@@ -76,7 +76,17 @@ This document describes an architecture for exposing transport protocol features
 Many application programming interfaces (APIs) to perform transport networking have been deployed, perhaps the most widely known and imitated being the BSD Socket {{POSIX}} interface (Socket API). The naming of objects and functions across these APIs is not consistent and varies depending on the protocol being used. For example, sending and receiving streams of data is conceptually the same for both an unencrypted Transmission Control Protocol (TCP) stream and operating on an encrypted Transport Layer Security (TLS) {{?RFC8446}} stream over TCP, but applications cannot use the same socket ```send()``` and ```recv()``` calls on top of both kinds of connections.
 Similarly, terminology for the implementation of transport protocols varies based on the context of the protocols themselves: terms such as "flow", "stream", "message", and "connection" can take on many different meanings. This variety can lead to confusion when trying to understand the similarities and differences between protocols, and how applications can use them effectively.
 
-The goal of the Transport Services architecture is to provide a flexible and reusable architecture that provides a common interface for transport protocols. As applications adopt this interface, they will benefit from a wide set of transport features that can evolve over time, and ensure that the system providing the interface can optimize its behavior based on the application requirements and network conditions, without requiring changes to the applications. This flexibility enables faster deployment of new features and protocols. It can also support applications by offering racing mechanisms (attempting multiple IP addresses, protocols, or network paths in parallel), which otherwise need to be implemented in each application separately (see {{racing}}).
+The goal of the Transport Services architecture is to provide a flexible
+and a reusable architecture that provides a common interface for transport protocols. An application uses the Transport Services system through an abstract Connection.
+This provides
+flexible connection establishment allowing an application to request or require a set of properties.
+The system selects one or more candidate each with equivalent protocol stacks.
+This set of candidates can be raced to identify
+an optimal combination of transport protocol instance such as TCP, UDP, or another transport, together with  configuration of parameters and
+interfaces. The Connection represents an object that, once established, can be used to send and receive messages.
+A Connection can be created from another Connection, by cloning, and then forms a part of a Connection Group that share properties.
```

Comment:
Something's grammatically weird here, with plural "share" and singular "Group". Perhaps:
```suggestion
A Connection can be created from another Connection, by cloning, and then forms a part of a Connection Group whose Connections share properties.
```